_data/projects/simde.yml

```diff
@@ -0,0 +1,13 @@
+name: SIMDe
+desc: Implementations of SIMD instruction sets for systems which do not natively support them.
+site: https://github.com/nemequ/simde
+tags:
+- C
+- SIMD
+- parallel
+- optimization
+- portability
+- performance
+upforgrabs:
+  name: good first issue
+  link: https://github.com/nemequ/simde/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

Comment:
@nemequ Please change this to https://github.com/nemequ/simde/labels/good%20first%20issue